app/views/subjects/show.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container">
```

Comment:
I think here will be better to show not a table, but something else
Or actually up to you

Reply:
i left the same for right now.